build-system/babel-config/unminified-config.js

```diff
@@ -45,6 +45,7 @@ function getUnminifiedConfig() {
   const unminifiedPlugins = [
     argv.coverage ? 'babel-plugin-istanbul' : null,
     replacePlugin,
+    './build-system/babel-plugins/babel-plugin-transform-remove-directives',
```

Comment:
This was needed because of how we were previously adding the wrapper to individual files before bundling. I think I can get rid of this in a follow up PR.

Reply:
Removing this in #32696.